src/ops/delete_objects.rs

```diff
@@ -64,6 +64,7 @@ pub async fn extract(ctx: &mut ReqContext<'_>) -> S3Result<DeleteObjectsRequest>
 }
 
 impl S3Output for DeleteObjectsOutput {
+    #[allow(clippy::shadow_unrelated)]
```

Comment:
why allow this clippy lint?

Reply:
The xml writer uses an "explict recursive" style. It is widely used among this codebase. `clippy::shadow_unrelated` conflicts with the api style. Fixing it needs pretty much changes.